common/src/main/java/io/quarkus/ts/openshift/common/util/AwaitUtil.java

```diff
@@ -45,7 +45,7 @@ public void awaitAppRoute() {
 
         System.out.println(ansi().a("waiting for route ").fgYellow().a(metadata.appName).reset()
                 .a(" to start responding at ").fgYellow().a(metadata.knownEndpoint).reset());
-        await().ignoreExceptions().atMost(5, TimeUnit.MINUTES).untilAsserted(() -> {
+        await().ignoreExceptions().atMost(10, TimeUnit.MINUTES).untilAsserted(() -> {
```

Comment:
I have to increase this timeout as it tends to spend more time downloading artifacts from artifactory.